ipatests/test_webui/test_navigation.py

```diff
@@ -171,6 +171,39 @@ def test_menu_navigation(self):
             self.assert_menu_item('ipaserver/trusts', False)
         self.navigate_by_menu('ipaserver/config', False)
 
+    def test_new_tab_navigation(self):
+        """
+        Test for PF#7137: [RFE]: Able to browse different links
+        from IPA web gui in new tabs
+
+        Test verifies whether opening target link in new tab
+        navigates to target (desired behaviour) compared to creation of copy of
+        current state of page on new tab (old behaviour).
+
+        Related: https://pagure.io/freeipa/issue/7137
+        """
+        self.init_app()
+        crubs = [
+            'identity/host',
+            'identity/user_search',
+            'identity/group_search',
+            'identity/idview',
+            'identity/automember',
+            'policy',
+            'policy/pwpolicy',
+            'authentication',
+            'network_services',
+            'ipaserver'
+        ]
+        for crub in crubs:
+            self.navigate_by_menu(crub, False)
+            initial_url = self.driver.current_url
+            rows = self.get_rows()
+            if len(rows) > 0:
+                self.navigate_to_row_record_in_new_tab(rows[-1])
+            else:
+                continue  # no clickable results returned
```

Comment:
You could move the assert to the condition block, so `else: continue` block would not be needed.

Reply:
Good point, moved